doc/build-openbsd.md

```diff
@@ -13,10 +11,10 @@ Run the following as root to install the base dependencies for building:
 
 ```bash
 pkg_add git gmake libevent libtool boost
+pkg_add qt5 # (optional for enabling the GUI)
```

Comment:
Got some errors while running this command: https://pastebin.com/raw/3qvFa07m

Environment details: 

[OpenBSD 6.7](https://cdn.openbsd.org/pub/OpenBSD/6.7/amd64/install67.iso)
1 GB RAM
20 GB Virtual Disk

Followed most of the instructions mentioned in one [article](https://medium.com/linux-gossip/openbsd-6-4-installing-a-seriously-underrated-os-in-a-virtual-machine-f5848ee5a25a) to setup OpenBSD VM.

Reply:
> It appears the error is due to running out of space in /usr/local/

Yes I will have to figure out how to resolve this before trying to build bitcoin core.

![image](https://user-images.githubusercontent.com/13405205/92873951-2fbb6900-f425-11ea-8b67-466491a739ae.png)

Reply:
If you want to try reinstalling OpenBSD, you could try manually partitioning only one partition instead. That way you'll have adequate space. https://nanxiao.me/en/create-only-root-partition-during-installing-openbsd/

Reply:
> If you want to try reinstalling OpenBSD, you could try manually partitioning only one partition instead. That way you'll have adequate space. https://nanxiao.me/en/create-only-root-partition-during-installing-openbsd/

This worked and I was able to add the required packages. However got some errors while running the `make` command:

![image](https://user-images.githubusercontent.com/13405205/93005312-6f767380-f53f-11ea-9e9f-ee6c76013878.png)

Reply:
I'm thinking you're running out of memory now. How much RAM have you allocated to the VM?

Reply:
Changing RAM from 1 GB to 3 GB partially resolved the issue. This time there was a different error after few minutes of compiling.

![image](https://user-images.githubusercontent.com/13405205/93009639-0de6f100-f5a1-11ea-8226-a442f9087d68.png)

Reply:
Are you trying to build master? I've been building v0.20.1 successfully.

Reply:
> This time there was a different error after few minutes of compiling.

The issue you're seeing is #19559. If you append `CC_FOR_BUILD=cc` to your `./configure` command, everything should work ok. Note that if you're building master, we also just merged a update to the libsecp256k1 subtree, which contains a fix for this issue. So if you `git pull` and rebuild (after cleaning) it may also disappear. Or, as mentioned above, you can checkout and build a specific tag.